### PR TITLE
User Rspec instead of minitest

### PR DESCRIPTION
**Pull request by apurvis**
@olleolleolle @kamui  - as discussed in #59 
this actually only took like 15-20 minutes to migrate with the help of a few regexes

### CODE REVIEW

File: spec/exponential_backoff_spec.rb [hunk 1]
```diff
@@ -40,46 +36,46 @@
...
+    expect(described_class.new(multiplier: 1).intervals).to eq([
       0.5244067512211441,
       0.607594682584082,
       0.5513816852888495,
     ])
   end
 
   it "generates intervals with a defined max interval" do
-    expect(subject.new(max_interval: 1.0, rand_factor: 0.0).intervals).must_equal([
+    expect(described_class.new(max_interval: 1.0, rand_factor: 0.0).intervals).to eq([
```
[the remark this answers is not in the document]
Comment (apurvis) [reply spec/exponential_backoff_spec.rb]: totally mysterious why travis won't respect [rubocop.yml](https://github.com/kamui/retriable/blob/master/.rubocop.yml#L31-L32)

[the remark this answers is not in the document]
Comment (kamui) [reply spec/exponential_backoff_spec.rb]: Try rebasing off master. I think hound uses a default rubocop config unless you specify an override.

[the remark this answers is not in the document]
Comment (apurvis) [reply spec/exponential_backoff_spec.rb]: it's only travis... when i run `rubocop` on the project locally it finds `.rubocop.yml` just fine and uses it

[the remark this answers is not in the document]
Comment (kamui) [reply spec/exponential_backoff_spec.rb]: Travis doesn't run rubocop, houndci runs it off the github repo.

[the remark this answers is not in the document]
Comment (apurvis) [reply spec/exponential_backoff_spec.rb]: yeah that fixed it; now the only hounds i get are for over 120 lines

**Comment by kamui** [comment]
Looks great, thanks for taking the time to switch this to rspec. I'll merge shortly after specs pass, unless you're still cleaning up.

**Comment by apurvis** [comment]
nah i'm good for now cleaning up wise... just being pedantic at this point.

**Comment by apurvis** [comment]
in other words, merge away

**Comment by apurvis** [comment]
see also: #62

**Pull request merged**